cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -6,13 +6,59 @@
 
    Description:
 
-   Create a hero molecule. See [GHE]/flapjack/Modules-V1/wiki/Heros
+   Create a hero molecule.
+
+   settings:             An object with the following options for settings.
+
+   REQUIRED:
```

Comment:
Idea: If we're going to do this what about passing in two objects instead so that the code is self-documenting as well? E.g. the method signature could be `macro render(settings, options={})` and then you'd have `options.background` and `options.links`.